src/app/pages/adoption-confirmation/adoption-confirmation.js

```diff
@@ -0,0 +1,137 @@
+import BaseView from '~/helpers/backbone/view';
+import {props} from '~/helpers/backbone/decorators';
+import {template} from './adoption-confirmation.hbs';
+
+function hookUpCalculator(tableBody) {
+    let studentCountEl = tableBody.querySelector('#student-count'),
+        rows = tableBody.querySelectorAll('tr');
+
+    function calculateRow(row, rowIndex) {
+        let tds = row.querySelectorAll('td'),
+            tPrice = tds[1].querySelector('input').value,
+            osInput = tds[0].querySelector('input'),
+            osPrice = osInput ? osInput.value : 0,
+            stuInput = tds[2].querySelector('input'),
+            stuCount = +(stuInput ? stuInput.value : tds[2].innerHTML);
```

Comment:
Since this is submitting a form, you should be able to listen to the `submit` event and glean all the data from that without needing to resort to all the DOM selectors.